app/Http/Controllers/ProductController.php

```diff
@@ -60,6 +75,15 @@ public function show($id)
      */
     public function edit($id)
     {
+        try {
+            $product = Product::findOrFail($id);
+
+            return view('pages.products.edit', [
+                'product' => $product
+            ]);
```

Comment:
```suggestion
            return view('pages.products.edit', compact('product'));
```
dùng cái này ngắn hơn nè